admin/adrs/0001-select-folder-structure.md

```diff
@@ -0,0 +1,22 @@
+# Selecting an Architecture/Folder Structure for our Toolkit
+
+## Context and Problem Statement
+
+Our group needed to decide on what file structure to use to start working on our project
+
+## Considered Options
+
+* IFrame Design
+  - Creating an Iframe for every component of our toolkit
+  - Similar to an Angular JS folder structure
+  - Very Modular
+  - Easy to compartmentalize 
+* React-like Design
+  - No separate HTML files, included in JS
+  - Cohesive and fewer files to maintain
+  - Less Setup
+
+## Decision Outcome
+
+Chosen option: React-like Design
+Decided against the IFrame deisgn because Iframe creates a new tab everytime and can lead to wasted resources. Hence, the react-like structure was chosen for now.
```

Comment:
Maybe it would be great to add that we decided to explore the structure instead of sticking to some certain structure.